### DISCUSSION TITLE
Pyphotometry acquisition board

### DISCUSSION THREAD

**Discussion by anushree-umu** [pyPhotometry/code#11] [Q&A] [answered]
Hello,
I am new to the world of Pyphotometry. We have a Thor labs system and  I am looking to get the pyPHotometry acquisition board for the collection of data.
As I understand there are only 2 analog ports. I was hoping to get info from red, green channel along with the isobestic point recording. That would then mean I will need 3 analog connections? Is there a way anyone has increased this capacity of the board or any other way I can simultaneously record 3 channels?
Thank you

**Comment by ThomasAkam** [comment]
Hi @anushree-umu,

The board only supports two analog inputs, however normally when you aquire an isosbestic control channel this is done using the same photoreciever as the green channel, but with different wavelength excitation light sources for the green and isosbestic channels.  In principle I think it would be possible to aquire a red, green and isosbestic channel using the pyphotometry hardware, using one analog input for the green + isosbestic channel's photoreciever, and the other analog input for the red channel.  However, you would need to use an additional external LED driver as you would need a total of 3 excitation LEDs and the board only has two LED drivers on it.  You could trigger this LED driver using one of the digital lines on the board, which are usually used as inputs but which could be used as outputs as they are connected directly to pins of the pyboard microcontroller.  Implementing this 3 channel acqusition mode would require some modification of the software which I would be happy to give advice on but don't have plans to do myself in the immediate future.

**Comment by ThomasAkam** [comment, marked as answer]
Hi @anushree-umu,

3 channel (e.g. red/green/isosbestic) acqusition using an external LED driver for the 3rd channel is now supported by the [version 1.0](https://github.com/pyPhotometry/code/releases/tag/v1.0) release.  For more information see the `3EX_2EM_pulsed` acqusition mode in the [docs](https://pyphotometry.readthedocs.io/en/latest/user-guide/graphical-user-interface/#acquisition-modes).
